source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.js

```diff
@@ -110,6 +110,22 @@ export default class ChangeSpendingPasswordDialog extends Component<Props> {
             messages.currentPasswordFieldPlaceholder
           ),
           value: '',
+          validators: [
+            ({ field }) => {
+              if (this.props.isSpendingPasswordSet) {
+                if (field.value !== '') {
+                  return [
+                    isValidSpendingPassword(field.value),
+                    this.context.intl.formatMessage(
+                      globalMessages.invalidSpendingPassword
+                    ),
+                  ];
+                }
+                return [false];
```

Comment:
@DeeJayElly to prevent default errors, add something like this
```
return [
        false,
        this.context.intl.formatMessage(messages.fieldIsRequired),
];
```

Reply:
Added